packages/ui-link/src/Link/index.tsx

```diff
@@ -79,12 +79,23 @@ class Link extends Component<LinkProps, LinkState> {
   }
   ref: Element | null = null
 
+  checkProps = () => {
+    const { variant, isWithinText } = this.props
+    if (variant && isWithinText) {
+      console.warn(
```

Comment:
Do not use bare `console`, use our `console` package

Reply:
Changed